tools/cxbe/Makefile

```diff
@@ -1,5 +1,16 @@
-INCLUDES := $(wildcard *.h)
-SRCS := $(wildcard *.cpp)
+INCLUDES := AlignPosfix1.h \
+	AlignPrefix1.h \
```

Comment:
nit: Weird indentation. Either start it with an empty `\` in the first line, then align all with a tab; or align with spaces to align to the first line.